instance/models/mixins/openedx_config.py

```diff
@@ -75,7 +75,8 @@ def _get_configuration_variables(self):
 
             # Set this to a string such as ".myinstance.org" to enable session sharing between LMS and the Studio.
             # We cannot do this on OC IM for security reasons (we don't want different *instances* to share cookies).
-            "EDXAPP_SESSION_COOKIE_DOMAIN": '',
+            "EDXAPP_SESSION_COOKIE_DOMAIN": '.{}'.format(self.instance.domain),
```

Comment:
During my testing, I found that `instance.domain` and `instance.internal_lms_domain` are the same for instances created using the default subdomains.

These seem to be used interchangeably thorough the code, do you know if these can take different values? I couldn't find one such example.

Reply:
@viadanna, based on my understanding, `instance.domain` maps to the internal LMS domain or the external LMS domain when it is defined. But I am not able to find out the exact location where it is assigned. Will have to check more.

Reply:
👍